engine/src/main/java/com/datatorrent/stram/plan/logical/LogicalPlanConfiguration.java

```diff
@@ -147,13 +896,16 @@ public String toString(Object pojo)
       ObjectMapper mapper = ObjectMapperFactory.getOperatorValueDeserializer();
       try {
         return mapper.writeValueAsString(pojo);
-      } catch (Exception e) {
+      } catch (IOException e) {
```

Comment:
Why are we catching an IOException and throwing a RunTimeException instead? This seems somewhat strange. Why not re-throw the IOException with a more helpful message?